modules/rhcos-about.adoc

```diff
@@ -84,8 +83,9 @@ For {op-system} systems, the layout of the `rpm-ostree` file system has the
 * `/var/lib/containers` is the graph storage location for storing container
  images.
 
-In {product-title}, the Machine Config Operator handles operating
-system upgrades. Instead of upgrading individual packages, as is done with yum
+In {product-title}, the Machine Config Operator handles
```

Comment:
@sferich888, will you please review this link as well?

Reply:
@vikram-redhat, you probably shouldn't merge it with the second link. It would be better to port the content and get it approved in context.